app/lib/actions/katello/host/erratum/applicable_errata_install.rb

```diff
@@ -13,7 +13,7 @@ def plan(host, errata_ids)
           end
 
           def humanized_name
-            if input[:hostname]
+            if input.nil?
```

Comment:
I think we still want to check for `input[:hostname]` (in addition to `input.nil?`) in which case I'd probably just roll `input.try(:[], :hostname)`.